lib/logstash/filters/parser.rb

```diff
@@ -84,7 +84,7 @@ def calculate_bigram_frequency(tokens, index)
     singlegram = tokens[index - 1]
     doublegram = "#{singlegram}^#{tokens[index]}"
 
-    if @gramdict.double_gram_dict.include?(doublegram) && @gramdict.single_gram_dict.include?(singlegram)
+    if @gramdict.double_gram_dict.key?(doublegram) && @gramdict.single_gram_dict.key?(singlegram)
```

Comment:
Slight differences in method naming between the standard hashmap and `lru_redux` that I'm accounting for here